### PR TITLE
Adding support for DB_Assembler on the new directory structure

### PR DESCRIPTION
**Pull request by wizzymore**
Adding support for DB_Assembler on the new directory structure.

After this PR i think this can be merged into the skeleton module

### CODE REVIEW

File: sql/auth/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+development.sql
```
Comment (BarbzYHOOL) [inline sql/auth/.gitignore]: why? (sorry i'm out of the loop, i need explanation on basic things probably)

**Comment by BarbzYHOOL** [comment]
it's to make the db assembler work with the new SQL structure of my PR ?

And if so, does it work without modifying the core bash script ?

**Comment by wizzymore** [comment]
1. Yes
2. It requires to have an update core `config.sh` as it stores the login credentials to the database.
The module `config.sh.dist` stores where the sql files are located at

**Comment by BarbzYHOOL** [comment]
thank you !

**Comment by Yehonal** [comment]
> it's to make the db assembler work with the new SQL structure of my PR ?
> 
> And if so, does it work without modifying the core bash script ?

can you refer your PR here please?

**Comment by BarbzYHOOL** [comment]
https://github.com/azerothcore/skeleton-module/pull/8 mister lazy @Yehonal  xD

**Pull request merged**